source/includes/mongodb-compatibility-table-cxx.rst

```diff
@@ -12,7 +13,18 @@
      - MongoDB 2.6
      - MongoDB 2.4
 
+   * - mongocxx 3.5.x
```

Comment:
I'd suggest dropping the ".x" from each version in this table. Also, other tables tend to report "Version X.Y" instead of repeating the driver name ("mongocxx" in this case). PHP is a notable exception to that because we have both a low- and high-level driver and it helps to differentiate.